sections/algorithmic_components.tex

```diff
@@ -701,11 +697,10 @@ \subsection{Astrometric Fitting}
 \label{sec:acAstrometricFitting}
 \subsubsection{Single CCD}
 \label{sec:acSingleCCDAstrometricFit}
-Used by AP, probably (RHL worries we might need full-visit)
 
-AP will need to do reasonably good astrometeric calibration on single frames in order to do the relative warping between the template and science images.  We have seen that the kernel matching algorithm can take out bulk astrometric errors up to a significan fraction of a pixel.  However, we would like to avoid making the kernel matching deal with astrometric errors.  Astrometric errors between the science and template coordinate systems should be less than 15mas.  We will use the internal reference catalog used in DRP as the reference catalog.  This will be based on astrometry from an external source and will be extended using high quality measurements on coadds from DRP.
+AP will need good astrometeric calibration on single frames in order to do the relative warping between the template and science images.  We have seen that the kernel matching algorithm can take out bulk astrometric errors up to a significant fraction of a pixel.  However, we expect subtraction performance to improve if astrometric errors are minimized.  Astrometric errors between the science and template coordinate systems should therefore be less than 15\,mas.  We expect to use the internal reference catalog used in DRP as the reference catalog.  This will be based on astrometry from an external source and will be extended using high quality measurements on coadds from DRP.
 
-We can project the per-chip coordinates into a flat (Gnomonic) projection in the focalplane.  This will take out the bulk of the optical distortion.  Next, we will use a matching algorithm like that outlined in \cite{tabur04}.  Once we match, a 2D polynomial solution will be fit to minimize the residual.
+We can project the per-chip coordinates into a flat projection in the focalplane.  This will take out the bulk of the optical distortion.  We will then match the science and template sources and fit a distortion model to minimize the residual positional offsets.
```

Comment:
It actually looks like we will not match in focalplane coordinates but on the surface of a unit sphere.  This has some nice properties and serves the same purpose of taking out the bulk optical distortion.  I say this based on the recent work by Chris M. (DMTN forthcoming).  I don't see a reason not to state this as the baseline right now.